src/redux/wishlist/wishlist.actions.js

```diff
@@ -0,0 +1,27 @@
+import {
+  SET_WISHLIST,
+  GET_WISHLIST,
+  ADD_ITEM_TO_WISHLIST,
+  REMOVE_ITEM_FROM_WISHLIST
+} from './wishlist.types';
+
+const getWishlist = () => ({
+  type: GET_WISHLIST
+});
+
+const setWishlist = (wishlistItems) => ({
+  type: SET_WISHLIST,
+  payload: wishlistItems
+});
+
+const addItemToWishlist = (item) => ({
+  type: ADD_ITEM_TO_WISHLIST,
+  payload: item
+});
+
+const removeItemFromWishlist = (itemId) => ({
+  type: REMOVE_ITEM_FROM_WISHLIST,
+  payload: itemId
+});
+
```

Comment:
consider adding wishlist-loading

Reply:
there is no any asynchronous logic yet

Reply:
Dmytro is right, for now, the wishlist is completely on client-side